src/App.tsx

```diff
@@ -45,11 +45,11 @@ const App = () => {
                 element={<ProposalPage />}
               />
               <Route
-                path="/:chainName/:guildId/create/:proposalType"
+                path="/:chainName/:guildId/create-proposal"

```

Comment:
I think this should be camelCase, or switch to kebab-case for the `allProposals` route.